about.html

```diff
@@ -69,6 +69,6 @@ <h3 class="clemson-orange-text">Educational Benefits of a CTF</h3>
     </div>
   </div>
   <h3 class="clemson-orange-text">When and Where?</h3>
-  <p>CUCTF will take place October 3rd, 2020, hosted at Clemson's most technologically advanced building: The Watt Center.</p>
+  <p>CUCTF 2020 will take place <strong>October 3rd, 2020</strong>. While normally hosted at Clemson University's Watt Family Innovation Center, <strong>this event will be hosted online</strong>. Stay tuned, the specific time of the event has yet to be announced! </p>
```

Comment:
By specific time are you meaning like the schedule? I would probably say something like "Stay tune, more information on competing remotely will be announced soon!"